src/python/pants/backend/python/subsystems/python_tool_base.py

```diff
@@ -155,6 +161,14 @@ def lockfile(self) -> str:
         """
         return cast(str, self.options.experimental_lockfile)
 
+    @property
+    def is_lockfile(self) -> bool:
```

Comment:
How about `lockfile_used`? `Black.is_lockfile()` is a bit of a misnomer: the subsystem itself is never a lockfile.